beaker.gemspec

```diff
@@ -48,6 +48,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'stringify-hash', '~> 0.0'
   s.add_runtime_dependency 'beaker-hiera', '~> 0.0'
   s.add_runtime_dependency 'beaker-hostgenerator'
+  s.add_runtime_dependency 'beaker-puppet', '~> 0.0'
```

Comment:
@er0ck yes. So far, we've started publishing beaker-libraries at the `0.1.0` version, and used general major version rules for eventually moving to `1.y`, so this is what I'm going for. It's not much of a precedent though, so I'm open to different viewpoints if you'd like to suggest a better way.

I assume at this point that moving from beaker including this gem to having it exist completely independently will be the time for the `1.y` cut, but we'll see what happens in between now and then.